sigma/plugins/fun/xkcd.py

```diff
@@ -4,15 +4,17 @@
 
 async def xkcd(cmd, message, args):
 
-    random_comic = True
     if args:
         try: int(args[0]) #args[0] is always a string
         except ValueError:
             await message.channel.send(None, embed=discord.Embed(title=':exclamation: Invalid number', color=0x993333))
             return
 
         comic_no = args[0]
-        random_comic = False
+
+    random_comic = (len(args) == 0)
+        # True if args is empty
+        # False otherwise
 
```

Comment:
Don't indent comments; put comments before line; these technically be on one line

Reply:
Fixed in 0f64245